common/devdoc/interlocked_hl_requirements.md

```diff
@@ -59,92 +59,84 @@ then `InterlockedHL_Add64WithCeiling` fails and returns `INTERLOCKED_HL_ERROR`.
 ###  InterlockedHL_WaitForValue
 
 ```c
-MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_WaitForValue, int32_t volatile_atomic*, address, int32_t, value, uint32_t, milliseconds)(INTERLOCKED_HL_OK, INTERLOCKED_HL_ERROR);
+MOCKABLE_FUNCTION_WITH_RETURNS(, INTERLOCKED_HL_RESULT, InterlockedHL_WaitForValue, int32_t volatile_atomic*, address_to_check, int32_t, value_to_wait, uint32_t, timeout_ms)(INTERLOCKED_HL_OK, INTERLOCKED_HL_ERROR);
 ```
 
-`InterlockedHL_WaitForValue` waits for the value at a given address to be equal to a target value.
-
-**SRS_INTERLOCKED_HL_01_002: [** If `address` is `NULL`, `InterlockedHL_WaitForValue` shall fail and return `INTERLOCKED_HL_ERROR`. **]**
-
-**SRS_INTERLOCKED_HL_01_003: [** If the value at `address` is equal to `value`, `InterlockedHL_WaitForValue` shall return `INTERLOCKED_HL_OK`. **]**
+`InterlockedHL_WaitForValue` waits for the value at a given address_to_check to be equal to the target value_to_wait.
```

Comment:
>address_to_check [](http://example.com/codeflow?start=60&length=16)

Now these need backticks, because they are actually referencing the parameters (with underscores et all). Note: before the change, address was used as noun: "waits for the value at a given address to be equal to a target value".

Reply:
Fixed.

Reply:
Actually now it sounds weird. I suggest one of the 2:
- Either revert the change for the synopsis and leave line 65 as it was "`InterlockedHL_WaitForValue` waits for the value at a given address to be equal to a target value."
- Or phrase as "`InterlockedHL_WaitForValue` waits for the value at the given `address_to_check` to be equal to the target `value_to_wait`."

Reply:
Fixed.